### PR TITLE
Task 1 but ticket 4

### PR DESCRIPTION
**Pull request by hedieh623**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,23 +1,28 @@
...
+import Comments from "./Components/Comments"
+import { Route, Routes } from "react-router-dom";
+import { useState } from "react";
+import { useEffect } from 'react';
+
+
+
 
 function App() {
+  const [articles, setArticles] = { useState };
```
Comment (Augs0) [inline src/App.js]: This isn't the correct way to use useState. It should be useState() with the default data inside of the round brackets e.g. useState([])

File: src/Components/Header.jsx [hunk 2]
```diff
@@ -0,0 +1,5 @@
+const Header = () => {
+  return <h1> Northcoders News</h1>;
```
Comment (Augs0) [inline src/Components/Header.jsx:2]: It would be good to use <header> around this h1

File: src/Components/Topics.jsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+
+const Topics = ({ articles }) => {
+  const [allTopics, setAllTopics] = useState([]);
+  useEffect(() => {
+    axios
+      .get(`https://my-fe-super-duper-cool-app.herokuapp.com/=${articles}`)
```
Comment (Augs0) [inline src/Components/Topics.jsx]: The use of articles as your prop here is very confusing in a topics component

**Review by Augs0 — CHANGES_REQUESTED** [review]
This is a good start, but there's some incorrect syntax and lots of components created that don't need to exist yet e.g. Comments. Try not to jump ahead to other tickets. Please fix the mistakes so we are able to see your articles rendered

**Comment by Augs0** [comment]
Please also add a screenshot of your plan so other mentors can see it :)

File: src/Components/Articles.jsx [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+import { useState } from "react";
+import axios from "axios";
+const Articles = () => {
+  const [articles, setArticles] = useState([]);
+  
+  useEffect(() => {
+    axios
+      .get(`https://my-fe-super-duper-cool-app.herokuapp.com/api/articles`)
+      .then(({ data }) => {
+        console.log(data.articles);
```
Comment (yalicee) [inline src/Components/Articles.jsx:11]: remove this console.log

File: src/Components/Articles.jsx [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+        setArticles(data.articles);
+      })
+      .catch((err) => {});
+  }, []);
+  return (
+    <main>
+      <ul>
+        {articles.map((article) => {
+          return <li key={article.article_id}>{article.title}</li>;
```
Comment (yalicee) [inline src/Components/Articles.jsx]: You haven't added anything more such as the author, date, votes or body

**Review by yalicee — CHANGES_REQUESTED** [review]
See the comments I have added. Also you need to delete all of the components you are not using

**Review by leekli — APPROVED** [review]
Hi Hedieh, I can see you've made the changes to this.

I'm technically happy to sign this off as it adheres to the task, however I would say it's not displayed in the best way it could it, at the moment it's a simple list of article titles, separated by date and author with commas. It's not the easiest to read.

You are free to merge this branch, however as you move to your next ticket I would encourage you to spend some time having a think about how to present the info to the user in a nice, easy to see way.

**Pull request merged**